app/src/main/java/com/iic/shopingo/ui/OnboardingActivity.java

```diff
@@ -39,6 +45,14 @@ public void call(Session session, SessionState sessionState, Exception e) {
   @Override
   protected void onResume() {
     super.onResume();
+
+    // For scenarios where the main activity is launched and user session is not null, the session state change notification
```

Comment:
If you bring your app to background and then return to foreground.
onResume is called and not onCreate.